### ISSUE TITLE
Can't install from pip

### ISSUE THREAD

**Issue by Marc-3d** [elhuhdron/pylibczi#6]
Hi, 

I am trying to install pylibczi in an anaconda environment with python version 3.10.4

I got the following error after runing "pip install pylibczi": 

  ERROR: Could not find a version that satisfies the requirement pylibczi (from versions: none)
  ERROR: No matching distribution found for pylibczi

Do I need any specific version of python?

EDIT: I was able to install it in an anaconda environment with python version 3.6.13. Is pylibczi incompatible with 3.10? Maybe a small change to include 3.10 in the "python version requirements" (I don't actually know how python packages work) solves it.

Thanks in advance

**Comment by elhuhdron** [comment]
Sorry, this repo has not been updated and I have not pushed any new builds in quite some time. As noted in the [README](https://github.com/elhuhdron/pylibczi/blob/master/README.md), please try the [aicspylibczi fork](https://github.com/AllenCellModeling/aicspylibczi) as it contains many new features that improve greatly on the original. I'm basically not currently maintaining this version. You might also consider [aicsimageio](https://github.com/AllenCellModeling/aicsimageio); it is more of a high level image io interface, and utilizes aicspylibczi for reading czi files.

**Comment by Marc-3d** [comment]
Thanks for you answer, I could install pylibczi in python 3.6.13 :) 

I am trying to install aicsimageio but I also get some errors along the way. I will keep trying...

**Comment by elhuhdron** [comment]
Yeah, it looks like the last [builds](https://pypi.org/project/pylibczi/#files) for pypi that I created were for python 3.4-3.7 (linux), 3.6-3.7 (windows) and 3.5-3.7 (macos). It's interesting to me that some people are still using my version :) I figured everyone had switched over to the aics libraries. But, seems there is still some interest in this super minimalist version, so I'll update the pip installs for newer python versions when I find some time.
